src/main/java/medicaldoctor/controllers/ControllerManager.java

```diff
@@ -19,12 +19,13 @@ public void addScreen(String screenName, Node node){
         screen.put(screenName,node);
     }
     
-    public Node getScrene(String screenName){
+    public Node getScreen(String screenName){
         return screen.get(screenName);
     }
     
-    public void load(String name, String fxmlFile) throws IOException{
-        FXMLLoader loader = new FXMLLoader(getClass().getClassLoader().getResource(fxmlFile));
+    public void load(String name) throws IOException{
+        String path = name+".fxml";
```

Comment:
?
i mean you can literally just use "xxx.fxml" as the hash key? or am i misunderstanding?

Reply:
you can, but it looks cleaner in the LookUp class 😃

Reply:
i guess